draft-looker-oauth-attestation-based-client-authentication.md

```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
 
 --- back
 
+# Additional Examples
+
+## Wallet Instance Attestation
+
+This non-normative example shows a client attestations used as an wallet instance attestation in the context of eIDAS 2.0, e.g. to secure a Type-1 configuration credential. The additional claims describe the wallet's device binding und user binding capabilities and the achievable level of assurance.
```

Comment:
If you mention eIDAS terms, like "Type-1 configuration", you also need to add a reference to the ARF.